app/helpers/job_invocation_output_helper.rb

```diff
@@ -29,7 +28,7 @@ def colorize_line(line)
     parts.each do |console_line|
       if console_line.include?('{{{format')
         if (color_index = console_line[/color:(\d+)/, 1]).present?
-          current_color = CONSOLE_COLOR[color_index]
+          current_color = CONSOLE_COLOR[color_index] || 'default'
```

Comment:
Setting a default value on a frozen hash isn't possible - so this is the best next option.